cypress/integration/app/recommendations/recommendations-list-paginator/recommendations-list-paginator.spec.ts

```diff
@@ -6,33 +6,4 @@ describe("Recommendations list paginator", () => {
   it("should allow me to visit the page", () => {
```

Comment:
Maybe just remove these files completely

Reply:
Re-running the jobs solves the issue for authorisation @CaiWillis